qiita_db/support_files/patches/72.sql

```diff
@@ -0,0 +1,67 @@
+-- January 4, 2019
+-- add external_job_id column to record mapping of Torque Job IDs to Qiita Job IDs.
+-- COMMENT ON COLUMN qiita.processing_job IS 'Store an external job ID (e.g. Torque job ID) associated this Qiita job.';
+
+ALTER TABLE qiita.processing_job ADD external_job_id varchar;
+COMMENT ON COLUMN qiita.processing_job.external_job_id IS 'Store an external job ID (e.g. Torque job ID) associated this Qiita job.';
+
+CREATE TABLE qiita.processing_job_resource_allocation
+(
+	name	varchar,
+	description	varchar,
+	type	varchar,
+	allocation	varchar
+);
+
+insert into qiita.processing_job_resource_allocation (name, description, type, allocation) values ('REGISTER', 'single-core-8gb', 'REGISTER', '-q qiita -l nodes=1:ppn=1 -l pmem=8gb -l walltime=50:00:00');
```

Comment:
Is this something that the plugins should be defining and Qiita be consuming? Seems to me that by defining these values here, the plugins become very dependent on qiita and vice-versa.

This is likely outside of the scope of this PR, but it would probably be handy to have the resource allocations for each job in a user-facing location.

Reply:
That's very true, it would be good to have at least a negotiation between qiita and plugins, regarding the kinds of resources they need, and what qiita can allow. I think getting this data into Qiita's database is a good start, and as you said, make it more useful in another PR.